### ISSUE TITLE
loading via requireJS in browser

### ISSUE THREAD

**Issue by jean-moldovan** [pauldijou/jwt-client#2]
Hi,
I'm getting `Uncaught Error: Script error for "global", needed by: jwt-client` when trying to load the browser version of jwt-client via requireJS.

Example:

```
    requirejs.config({
      paths: {
        'jwt-client': 'bower_components/jwt-client/jwt-client'
    });

    require(['jwt-client'], function (jwt) {
      console.log(jwt);
    });
```

Is there any workaround for that ?

**Comment by pauldijou** [comment]
Should be fixed in 0.1.3. Totally my fault, not a big user of requirejs and never tested it with this lib.

**Comment by jean-moldovan** [comment]
Thanks, v0.1.3 not available via bower though :smile:

**Comment by pauldijou** [comment]
Yeah... huge fail again... Sorry about that. Just released 0.1.4, hope it's good this time!

**Comment by jean-moldovan** [comment]
:+1:  Seems to be fine now.

Thanks for quick reply !
